README.md

```diff
@@ -14,18 +14,103 @@
+> - [Conflations artifacts](https://github.com/linebender/vello/issues/49).
+> - [GPU memory allocation strategy](https://github.com/linebender/vello/issues/366)
+
+## Motivation
+
+Vello is meant to fill the same place in the graphics stack as other vector graphics renderers like [Skia](https://skia.org/), [Cairo](https://www.cairographics.org/), and its predecessor project [Piet](https://github.com/linebender/piet).
+On a basic level, that means it provides tools to render shapes, images, gradients, text, etc, using a PostScript-inspired API, the same that powers SVG files and [the browser `<canvas>` element](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D).
+
+Vello's selling point is that it gets better performance than other renderers by better leveraging the GPU.
+In traditional PostScript renderers, some steps of the render process like sorting and clipping either need to be handled in the CPU or done through the use of intermediary textures.
+Vello avoids this by using prefix-sum algorithms to parallelize work that usually needs to happen in sequence, so that work can be offloaded to the GPU with minimal use of temporary buffers.
+
+
+## Getting started
+
+Vello is meant to be integrated deep in UI render stacks.
+While drawing in a Vello scene is easy, actually rendering that scene to a surface requires setting up a wgpu context, which is a non-trivial task.
+
+To use Vello as the renderer for your PDF reader / GUI toolkit / etc, your code will have to look roughly like this:
+
+```rust
+// Initialize wgpu and get handles
+let device: wgpu::Device = ...;
+let queue: wgpu::Queue = ...;
+let surface: wpgu::Surface<'_> = ...;
+let texture_format: wgpu::TextureFormat = ...;
+let mut renderer = Renderer::new(
+      &device,
+      RendererOptions {
+         surface_format: Some(texture_format),
+         use_cpu: false,
+         antialiasing_support: vello::AaSupport::all(),
+         num_init_threads: NonZeroUsize::new(1),
+      },
+).expect("Failed to create renderer");
+
+// Create scene and draw stuff in it
+let mut scene = vello::Scene::new();
+scene.fill(
+   vello::peniko::Fill::NonZero,
+   vello::Affine::IDENTITY,
+   vello::Color::rgb8(242, 140, 168),
+   None,
+   &vello::Circle::new((420.0, 200.0), 120.0),
+);
+
+// Draw more stuff
+scene.push_layer(...);
+scene.fill(...);
+scene.stroke(...);
+scene.pop_layer(...);
+
+// Render to your window/buffer/etc.
+let surface_texture = surface.get_current_texture()
+   .expect("failed to get surface texture");
+vello::block_on_wgpu(
+      &device,
+      renderer
+         .render_to_surface_async(
+            &device,
+            &queue,
+            &scene,
+            &surface_texture,
+            &render_params,
+         ),
+).expect("Failed to render to surface");
+surface_texture.present();
+```
+
+See the [`examples/`](examples) folder to see how that code integrates with frameworks like winit and bevy.
```

Comment:
Relative links in `README.md` are not great, because they will stop working when this file is rendered on [crates.io](https://crates.io/) or [docs.rs](https://docs.rs/). I think we should convert all relative links in `README.md` to absolute links. So this `examples` directory link here, but also a bunch of other directory links in the *Integrations* section.

Reply:
Ah, yeah, that makes sense.